tests/functional/DatadirScenarioTest.php

```diff
@@ -122,7 +122,7 @@ private static function getSchema2Config(): array
 
     protected function getScript(): string
     {
-        return $this->getTestFileDir() . '/../../../src/run.php';
+        return $this->getTestFileDir() . '/../../src/run.php';
```

Comment:
Subor bol presunuty `tests/functional/scenario1/DatadirScenarioTest.php` → `tests/functional/DatadirScenarioTest.php`.

Kedze to nezodpovedalo namespacom a nova verzia Composer autoload ho nevedela najst.